en-US/Conductor_Handbook/themes/pdf-theme.yml

```diff
@@ -0,0 +1,42 @@
+extends:
+  - default
+page:
+  size: Letter
```

Comment:
This will possibly differ depending on the language, since people in other countries may want other formats.